pkgs/tools/misc/tmux-sessionizer/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, openssl
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tmux-sessionizer";
+  version = "v0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jrmoulton";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-FSh3ww5KpktNSvqB3kcIaTxNHypwsVTHze0mgBtuJQE=";
+    };
+
+  cargoSha256 = "sha256-VMdXlBRzM+p7EsU3Wh9Lpg4V1KACX4DUNTh4ARlDwOc=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
```

to fix build on Darwin:
```suggestion
  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ Security ];
```